frontend/app/routes/$lang+/_public+/apply+/index.tsx

```diff
@@ -88,15 +88,19 @@ export default function ApplyIndex() {
 
   function handleSubmit(event: FormEvent<HTMLFormElement>) {
     event.preventDefault();
-    if (captchaRef.current) {
-      const formData = new FormData(event.currentTarget);
-      const response = captchaRef.current.getResponse();
-      formData.set('h-captcha-response', response);
-      fetcher.submit(formData, { method: 'POST' });
 
-      captchaRef.current.resetCaptcha();
+    const formData = new FormData(event.currentTarget);
+    try {
+      if (captchaRef.current) {
+        const response = captchaRef.current.getResponse();
+        formData.set('h-captcha-response', response);
+        captchaRef.current.resetCaptcha();
```

Comment:
Should this maybe be in a `finally` so it is always executed?

Reply:
@nicholas-ly -- since you're not here today, I will make this change and push a commit.